include/alpaka/atomic/AtomicStdLibLock.hpp

```diff
@@ -23,6 +23,8 @@
 
 #include <alpaka/atomic/Traits.hpp>
 
+#include <alpaka/core/BoostPredef.h>
```

Comment:
suffix to hpp: `<alpaka/core/BoostPredef.hpp>`

Reply:
done